torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9636,7 +9642,9 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            skips=(
                # Tests that assume input tensor has a meaningful effect on output tensor
                DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_variant_consistency_eager'),
-               DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_neg_view'),
+               DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_neg_view', device_type='cpu'),
+               DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_neg_view', device_type='cuda',
```

Comment:
So the test should fail on cuda unless we're running with TEST_WITH_TORCHINDUCTOR 🤔 that seems a bit odd

Reply:
Whatever the cause was, it seems to have gone away after rebasing (i.e. the test now uniformly fails regardless of `TEST_WITH_TORCHINDUCTOR`...)